swagger-spec/nodes/registrations_list.yaml

```diff
@@ -182,3 +182,64 @@ get:
               node_license:
             type: registrations
             id: esa63
+
+# /nodes/{node_id}/registrations/
+post:
+  summary: Create a registration
+  description: >-
+    Initiate a registration of the current draft registration.
+
+    Draft registrations contain the supplemental registration questions that accompany a registration.
+    A registration is a frozen version of the project that can never be edited or deleted, but can be withdrawn.
+
+
+    Your original project remains editable but will now have the registration linked to it.
+
+    #### Permissions
+
+    Only project administrators may view create registrations.
+
+    #### Required
+
+    Required fields for creating a registration include:
+
+
+    &nbsp;&nbsp;&nbsp;&nbsp`type`
```

Comment:
Any way to tab this without typing &nbsp...? Locally, this still looks like &nbsp.